portal/src/graphql/portal/ForgotPasswordSettings.tsx

```diff
@@ -5,23 +5,28 @@ import cn from "classnames";
 import deepEqual from "deep-equal";
 import produce from "immer";
 
+import { AppAndEmailSmsTemplatesConfigUpdater } from "./mutations/updateAppAndEmailSmsTemplatesMutation";
 import CodeEditor from "../../CodeEditor";
 import ButtonWithLoading from "../../ButtonWithLoading";
 import NavigationBlockerDialog from "../../NavigationBlockerDialog";
 import { setFieldIfChanged, clearEmptyObject } from "../../util/misc";
-import { PortalAPIAppConfig, PortalAPIApp } from "../../types";
+import { PortalAPIAppConfig, PortalAPIEmailAndSmsTemplates } from "../../types";
 
 import styles from "./ForgotPasswordSettings.module.scss";
-import TodoButtonWrapper from "../../TodoButtonWrapper";
+
+export const FORGOT_PASSWORD_EMAIL_HTML_TEMPLATE_NAME =
```

Comment:
Seems like there is a 1-1 relation between template item type and template name? I suggest we group them into a map, eg.

```typescript
export const TEMPLATE_NAME: Record<keyof PortalAPIEmailAndSmsTemplates, string> = {
    "emailHtml": "forgot_password_email.html",
    "emailText": "forgot_password_email.txt",
    "smsText": "forgot_password_sms.txt"
}
```